app/grandchallenge/notifications/signals.py

```diff
@@ -45,3 +53,27 @@ def create_post_action(sender, *, instance, created, **_):
         action.send(
             sender=instance.poster, verb="replied to", target=instance.topic,
         )
+
+
+@receiver(post_save, sender=Action)
+def create_notification(*, instance, **_):
+    if instance.target:
+        follower_group = followers(instance.target)
+        for follower in follower_group:
+            # only send notifications to followers other than the poster
+            if follower != instance.actor:
+                Notification(user=follower, action=instance).save()
```

Comment:
Just FYI - `Notification.objects.create(...)` is preferred if you're doing this in one step.